docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -11,60 +11,66 @@ Usage
 
 .. code-block:: cppkokkos
 
-    Kokkos::RangePolicy<>(begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(begin, end, args...)
-    Kokkos::RangePolicy<>(Space(), begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(Space(), begin, end, args...)
+    Kokkos::RangePolicy<...>(begin, end)
+    Kokkos::RangePolicy<...>(begin, end, ChunkSize(cs))
+    Kokkos::RangePolicy<...>(Space(), begin, end)
+    Kokkos::RangePolicy<...>(Space(), begin, end, ChunkSize(cs))
```

Comment:
Does `Kokkos::RangePolicy<...>` take any more function parameters than those shown on line 17?  If so, please list them, as they're part of the interface.  Also, please re-organize a bit  to have the "* Template arguments are all optional and can be given in arbitrary order" section to come immediately after your summary of the interface.